woocommerce/woocommerce-android
# [Tablet support M3] Part 2 – Toolbar refactoring

Closes: #10615 


### Description

This PR refactors toolbar implementation in order creation screen. We are switching from MainActivity AppBarLayout to local Toolbar components to support two-pane screen mode with separate titles, menus and navigation icons.

### Testing instructions

* Create new order and observe new toolbar layout on large screens based on the designs – 6kkCwI9VOEYcidp6UT5bln-fi-192%3A25891

* On the standard screen, the toolbar should look as before, however, the collapsing app bar layout behavior is removed.

Note that the toolbar of product selector fragment will be removed in the next PR.

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4527432/0d095fec-6ee9-4876-9ea9-ee27026481fb
